assets/css/variables.css

```diff
@@ -0,0 +1,35 @@
+:root {
+  /* font */
+  --primary-font: "open-sans";
+  --thin-font: 0.5em;
+  --normal-font: 1em;
+  --bold-font: 1.5em;
+  --bolder-font: 3em;
+  
+  /* spacing */
+  --spacing: 8px;
+  --spacing-xsmall: calc(var(--spacing));
+  --spacing-small: calc(var(--spacing) * 2);
+  --spacing-large: calc(var(--spacing) * 3);
+  --spacing-xlarge: calc(var(--spacing) * 4);
+  --spacing-xxlarge: calc(var(--spacing) * 5);
+
+  /* sizing */
+  --sizing: 10px;
```

Comment:
are they using for font size? if yes please change unit to em from px. keep in mind by changing the unit you need change the values as well. for example not 10em.

Reply:
I used them for border radius not font size.